libraries/testkit/user.py

```diff
@@ -143,6 +143,20 @@ def add_doc(self, doc_id=None, content=None, retries=False):
 
         return doc_id
 
+    def wait_until_db_online(self):
+        db_url = self.target.url + "/" + self.db + "/"
+        while True:
+            log.debug("DB [{}] waiting until online".format(self.db))
+            resp = self._session.get(db_url, timeout=settings.HTTP_REQ_TIMEOUT)
+            resp.raise_for_status()
```

Comment:
Good point

It should probably only try a limited amount of times before giving up too